modules/identity/pages/guest-user.adoc

```diff
@@ -59,6 +59,11 @@ Restart Bonita server.
 . If your configuration is correct you should be able to access your public application directly with the URL `http://<host>:<port>/bonita/apps/<app token>` without being logged in first. +
 You are done.
 
+[NOTE,caption=UI Builder]
+====
+If you are using the UI Builder, you need to recreate the UI Builder production container to be able to use the guest user access..
```

Comment:
**suggestion (if-minor):** I think it can be a good idea to add the link to the UIB production packaging here

Reply:
there is double points at the end 👀

Reply:
A little suggestion.
add "Make sure that your public apps are among packaged apps". no need it's seems already clear. But I see that having something like that could be helpful to the users.